conf/defaults.yaml

```diff
@@ -259,6 +259,10 @@ topology.disruptor.batch.size: 100
 topology.disruptor.batch.timeout.millis: 1
 topology.disable.loadaware: false
 topology.state.checkpoint.interval.ms: 1000
+topology.metrics.aggregate.per.worker: false
```

Comment:
@harshach 
Set this to 'true' breaks current metrics consumer implementations, since it delegates aggregation to metrics consumer by passing list of values. We need to guide the way how to aggregate these values to well-known metrics consumer implementations.

Btw, we should try avoiding to address current metrics feature and start re-designing new metrics feature. To tell the truth, this feature is actually closer to a hot-fix instead of improvement. 
Even I found the design issue while implementing this. Current metrics feature doesn't allow us to aggregate the values, since we can't add the metadata (which function we need to apply to aggregate) to metrics. Note that we expand metrics from value of metric, so adding metadata to metric doesn't make sense. Metric shouldn't use Object as value, especially some types are intended to be expanded.